### DISCUSSION TITLE
Feature Request: Use episode number from title

### DISCUSSION THREAD

**Discussion by ghost** [NNTmux/newznab-tmux#1278] [Ideas]
A lot of shows won't get tagged with an episode number which in turn makes it now show up in API searches or on the TV Series pages when that show can't be found on TVDB etc. at the time of indexing.

I suggest that if it fails we try to pluck the number from the title "S01E01".

**Comment by DariusIII** [comment]
I will have a look at the code and see what can be done to improve matching.

**Comment by ghost** [comment]
Oh, to clarify: It does not get an episode number because often at time of indexing the episode isn't added to TVDB, TVMaze etc. yet.

But if we fill in the episode number anyway, at least the API can then find it.

**Comment by ghost** [comment]
@DariusIII Any idea on how this might easily be implemented? Since the season/episode number is not in the releases table we can't easily add it without creating an episode in tv_episodes.

We could regex the season & episode out of the search name and write it into the XML response if tv_episode <= 0. Would be some extra CPU time though on every API request.
